src/windows.rs

```diff
@@ -124,6 +127,43 @@ pub fn list() -> Result<Vec<TrashItem>, Error> {
     }
 }
 
+pub fn metadata(item: &TrashItem) -> Result<TrashItemMetadata, Error> {
+    ensure_com_initialized();
+    unsafe {
+        let id_as_wide: Vec<u16> = item.id.encode_wide().chain(std::iter::once(0)).collect();
```

Comment:
I don't want to burden you, but I'd love it if there was a `to_wide_path(&OsStr) -> Vec<u16>` or something that replaces this code here and in 5 other places in this file.